### ISSUE TITLE
[Feature]: Make token context menu accessible through selection & impersonted panels

### ISSUE THREAD

**Issue by bubblobill** [RPTools/maptool#4086]
### Describe the Problem

Right-clicks present accessibility issues for some users, e.g. those using a touch UI.

### The Solution you'd like

Context menus could be presented as menus in contextually relevant panels.
For example, in the selection panel, a button to open the context menu within the panel itself.
This allows a single-click pointer event to activate the menu which would be friendlier. Ideally it would also have larger print for easier navigation.
Relates to #3371

Mockup below

### Alternatives that you've considered.

Rolling my own frame that offers similar functionality for those features supported by macro functionality.

### Additional Context

![touchSelected](https://github.com/RPTools/maptool/assets/45483160/153e9ed3-baac-48e3-9c3a-3e7825a8da53)

**Comment by bubblobill** [comment]
I've changed my mind. This should be a menu item _Token_ between View and Tool. The other features would be nice but accessibility-wise it should get into the standard menus first.
